### DISCUSSION TITLE
Statistical test for distribution fitting

### DISCUSSION THREAD

**Discussion by hadarshavit** [StatMixedML/XGBoostLSS#76] [Q&A]
Thanks for the great package!
Currently, when fitting data to a distribution only the NLLH loss is returned.
I would like to have some statistical test result as well, like the Kolmogorov–Smirnov test.
Any hint how I can get that?

Thanks!

**Comment by StatMixedML** [comment]
@hadarshavit Thanks for your interest in the project.

> I would like to have some statistical test result as well, like the Kolmogorov–Smirnov test.

Can you please provide an example of what you want to do exactly. I suppose you want to use the KS for comparing the fitted vs. the empirical distribution?

**Comment by hadarshavit** [comment]
Yes, given a fitted distribution, we currently get the NLLH loss. I would like to have the option to the the KS pvalue result as well.

**Comment by StatMixedML** [comment]
> Yes, given a fitted distribution, we currently get the NLLH loss. I would like to have the option to the the KS pvalue result as well.

Do you refer to an unconditional fit or to a fitted model including features? I suppose it is the former one, since the parameters for a fitted model are functions of the input features. So each observation has its own distribution.
